### DISCUSSION TITLE
docs/getting-started/quick-start/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#205] [Page comments]
# docs/getting-started/quick-start/

https://scratchaddons.com/docs/getting-started/quick-start/

**Comment by Fulcrum-19** [comment]
Hi! I really like Scratch Addons as I got it on my PC a while back. I'm trying to get it on my phone's chrome browser right now, but I'm not sure how to. Can I get some help?

**Reply by WorldLanguages** [reply]
Hello, see: https://scratchaddons.com/docs/faq/#can-i-use-scratch-addons-on-a-mobile-device
